tests/containers/forms/CreateCategoryForm.test.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import CreateCategoryForm from '../../../src/containers/forms/CreateCategoryForm';
+
+const successMessage = {
+  type: 'success',
+  text: 'Category created Successfully',
+};
+
+const infoMessage = {
+  type: 'info',
+  text: 'Sending ...',
+};
+
+const defaultState = {
+  name: '',
+  value: '',
+  description: '',
+  errors: {},
+};
+
+describe('<CreateCategoryForm />', () => {
+  let wrapper;
+  let mounted;
+  beforeEach((() => {
+    wrapper = shallow(<CreateCategoryForm.WrappedComponent
+      closeModal={() => { }}
+      createCategory={() => { }}
+    />);
+  }));
+
+  it('should render withour crashing', () => {
+    expect(wrapper).toHaveLength(1);
+  });
+
+  it('should show the <SingleInput/> components when they have loaded', () => {
+    wrapper.setState({ name: 'Test', value: '25' });
+    expect(wrapper.find('SingleInput').length).toEqual(2);
+  });
+
+  it('should show the <TextArea/> component when it has loaded', () => {
+    expect(wrapper.find('TextArea').length).toEqual(1);
+  });
+  it('should show the <Button/> component when it has loaded', () => {
+    wrapper.setState({ loading: false });
+    const inputComponent = wrapper.find('Button');
+    expect(inputComponent.length).toEqual(2);
+  });
+
+  it('should reset state on receipt of success message', () => {
+    mounted = mount(<CreateCategoryForm.WrappedComponent
```

Comment:
Initialize `mounted` at the beginning just like you've done with `wrapper` to avoid repetition. You can use `setProps` function to modify props.

Reply:
@Elbertbiggs360 Resolved?

Reply:
yes it is